apps/voting/scripts/buidler-hooks.js

```diff
@@ -0,0 +1,61 @@
+let pct16
+
+let tokens, accounts
+
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
+
+module.exports = {
+  postDao: async function ({ _experimentalAppInstaller, log }, bre) {
+    const bigExp = (x, y) =>
+      bre.web3.utils
+        .toBN(x)
+        .mul(bre.web3.utils.toBN(10).pow(bre.web3.utils.toBN(y)))
+    pct16 = (x) => bigExp(x, 16)
+
+    // Retrieve accounts.
+    accounts = await bre.web3.eth.getAccounts()
+
+    // Deploy a minime token an generate tokens to root account
+    const minime = await _deployMinimeToken(bre)
+    await minime.generateTokens(accounts[1], pct16(100))
```

Comment:
If this is for the local development environment, let's also mint some tokens for `accounts[0]` (perhaps even more or entirely replacing `accounts[1]`, so people can pass votes automatically).